utils/user-product-helper.ts

```diff
@@ -9,7 +9,7 @@ export function getProductInfo (config, type: ProductCode): ProductI {
     case ProductCode.BUSINESS:
       return {
         image: 'BCRS_dashboard_thumbnail_image.jpg',
-        link: config?.authURL || 'link_not_configured',
+        link: config?.myBusinessRegistryDashboard || 'link_not_configured',
```

Comment:
This is the fix...

from `https://dev.bcregistry.ca/business/auth/`

to `https://dev.bcregistry.ca/business/auth/account/business`.

Reply:
Looks good 👍